app/src/routes/ny.$id.inntekt-og-refusjon.tsx

```diff
@@ -1,5 +1,138 @@
-import { createFileRoute } from "@tanstack/react-router";
+import { ArrowLeftIcon, ArrowRightIcon } from "@navikt/aksel-icons";
+import {
+  BodyLong,
+  Button,
+  Heading,
+  Radio,
+  RadioGroup,
+  ReadMore,
+  VStack,
+} from "@navikt/ds-react";
+import { useSuspenseQuery } from "@tanstack/react-query";
+import { createFileRoute, Link } from "@tanstack/react-router";
+import { format } from "date-fns";
+import { nb } from "date-fns/locale";
+
+import {
+  forespørselQueryOptions,
+  personinfoQueryOptions,
+} from "~/api/queries.ts";
+import { Inntekt } from "~/features/skjema-moduler/Inntekt.tsx";
+import { InformasjonsseksjonMedKilde } from "~/features/skjema-moduler/PersonOgSelskapsInformasjonSeksjon.tsx";
+import { Fremgangsindikator } from "~/features/skjema-moduler/Skjemafremgang.tsx";
+import type { ForespørselEntitet } from "~/types/api-models.ts";
+import { capitalizeSetning, leggTilGenitiv } from "~/utils.ts";
 
 export const Route = createFileRoute("/ny/$id/inntekt-og-refusjon")({
-  component: () => <div>Hello /ny/$id/inntektsmelding!</div>,
+  component: InntektOgRefusjon,
 });
+
+function InntektOgRefusjon() {
+  const { id } = Route.useParams();
+  const forespørsel = useSuspenseQuery(forespørselQueryOptions(id)).data;
+
+  return (
+    <section className="mt-6">
+      <form className="bg-bg-default px-5 py-6 rounded-md flex gap-6 flex-col">
+        <Heading level="3" size="large">
+          Inntekt og refusjon
+        </Heading>
+        <Fremgangsindikator aktivtSteg={2} />
+        <ForeldrepengePeriode forespørsel={forespørsel} />
+        <Inntekt forespørsel={forespørsel} />
+        <UtbetalingOgRefusjon />
+        <Naturalytelser />
+        <div className="flex gap-4 justify-center">
```

Comment:
Skulle vi endret rekkefølgen på knappene, og hatt en `flex-col-reverse` klasse? Da blir rekkefølgen for tabbingen mer "riktig"